extensions/amp-analytics/0.1/vendors/bluetriangle.json

```diff
@@ -0,0 +1,39 @@
+{
+  "vars": {
+    "siteID": "",
+    "pageName": "",
+    "trafficSegment": "",
+    "pageType": "AMP Default",
+    "guid": "${clientId(bluetriangle_guid)}",
+    "sessionID": "${clientId(bluetriangle_sessionid)}"
+  },
+  "requests": {
+    "host": "https://d.btttag.com",
+    "endpoint": "${host}/btt.gif?",
+    "pageview": "${endpoint}?gID=${guid}&sID=${sessionID}&siteID=${siteID}&pageName=${pageName}&txnName=${trafficSegment}&thisURL=${canonicalUrl}&RefURL=${documentReferrer}&dom=${navTiming(domLoading,domComplete)}&domContentLoadedEnd=${navTiming(domContentLoadedEventEnd)}&domContentLoadedStart=${navTiming(domContentLoadedEventStart)}&domInteractive=${navTiming(domInteractive)}&domLoading=${navTiming(domLoading)}&domainLookupStart=${navTiming(domainLookupStart)}&elementCount=0&fByte=${navTiming(requestStart,responseStart)}&fCPnt=${firstContentfulPaint}&fetchStart=${navTiming(fetchStart)}&firstPaint=0&fmp=0&fps=0&fps_tti=0&loadEventStart=${navTiming(loadEventStart)}&lt_tti=0&nst=${navTiming(navigationStart)}&orderTND=0&pSz=0&pWSz=0&pageValue=&pgTm=${navTiming(navigationStart, loadEventStart)}&redirectCount=${navRedirectCount}&redirectStart=${navTiming(redirectStart)}&requestStart=${navTiming(requestStart)}&responseStart=${navTiming(responseStart)}&secureConnectionStart=${navTiming(secureConnectionStart)}&ssl=${navTiming(secureConnectionStart, responseStart)}&tcp=${navTiming(connectStart, connectEnd)}&top=0&totRD=0&tti=0&tul=0&unloadEventStart=${navTiming(unloadEventStart)}&largestContentfulPaint=${largestContentfulPaint}&cumulativeLayoutShift=${cumulativeLayoutShift}&firstInputDelay=${firstInputDelay}&wcd=0&eT=1&navigationType=1"
+  },
+  "transport": {
+    "beacon": false,
+    "xhrpost": false,
+    "image": true
+  },
+  "triggers": {
+    "trackPageview": {
+      "on": "visible",
+      "request": "pageview"
+    }
+  },
+  "extraUrlParams": {
+    "CN1": "0",
+    "CN2": "0",
+    "CN3": "0",
+    "CN4": "0",
+    "CN5": "0",
+    "CN6": "0",
+    "CN7": "0",
+    "CN8": "0",
+    "CN9": "0",
+    "CN10": "0",
+    "CSID": "0"
+  }
```

Comment:
Since you define these variables as `extraUrlParams`, they will appear in every request.

If this is the behavior you want, then you need to append these parameters to each requestUrl in `extensions/amp-analytics/0.1/test/vendor-requests.json`:

i.e. for `host`:
```
https://d.btttag.com?CN1=0&CN2=0&CN3=0&CN4=0&CN5=0&CN6=0&CN7=0&CN8=0&CN9=0&CN10=0&CSID=0
```